src/main/java/com/base/site/controllers/DailyLogController.java

```diff
@@ -42,17 +42,18 @@
     private final String DAILY_LOG = "dailyLog";
     private final String REDIRECT = "redirect:/";
 
-    @RequestMapping("/dailyLog")
-    public String dailyLog(DailyLog dailyLog, Model model,@Param("keyword") String keyword) {
+    @RequestMapping({"/dailyLog", "/dailyLog/{date}"})
+    public String dailyLog(DailyLog dailyLog, Model model,@Param("keyword") String keyword, @PathVariable(required = false, value = "date") String dateString) {
+        log.info("  get mapping DailyLog is called");
+        LocalDate date = dateString == null ? LocalDate.now() : LocalDate.parse(dateString);
```

Comment:
Wasn't this task part of the calendar function? We are going to have big merge conflicts.

Reply:
vi havde denne function før du begyndte på calender, jeg har bare samlet den under dailylog istedet for at have 2 getmapping funktioner på det. men vi kan lige snakke om det kl. 10